core/src/main/java/nl/nn/adapterframework/jdbc/JdbcFacade.java

```diff
@@ -301,6 +302,7 @@ public void setAuthAlias(String authAlias) {
 	public String getAuthAlias() {
```

Comment:
Waarom is er een `getDatasourceName` en een `getDataSourceNameToUse`?
Er moet een check in de configure zitten die controleert dat getDatasourceName niet leeg is.
`getDataSourceNameToUse` wordt dan overbodig en kan weg.